common/core/nesteddisplay.cpp

```diff
@@ -22,13 +22,78 @@
 #include <sstream>
 
 #include <hwctrace.h>
```

Comment:
Please fix your commit messaage. Check the format here:
https://github.com/intel/IA-Hardware-Composer/wiki/Contributions